### ISSUE TITLE
Should metrics description be a full sentence?

### ISSUE THREAD

**Issue by reyang** [open-telemetry/opentelemetry-specification#2975]
OpenMetrics provided such example: https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#overall-structure

![image](https://user-images.githubusercontent.com/17327289/203387202-fb2d9634-4591-4503-9e8e-ef87875b610d.png)

The current [metrics semantic convention](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#http-server) is different (and inconsistent):

![image](https://user-images.githubusercontent.com/17327289/203387432-85b03106-6366-4c44-9847-e9e57cdd2f78.png)

Ask:
1. Make it consistent at least for the semantic conventions that are in scope for the initial stable release.
2. Align with OpenMetrics, description should have the first letter uppercase, and end with `.`.

**Comment by carlosalberto** [comment]
Looks like a good, non-polemical improvement to make.

**Comment by trask** [comment]
@reyang do you think this is a blocker for HTTP stability?

**Comment by reyang** [comment]
I don't think so, worst case I'll send a small PR just to fix the HTTP metrics descriptions (to align with OpenMetrics convention). Assigning this to myself.

**Comment by reyang** [comment]
Update: I've scanned through the HTTP metrics semantic conventions and confirmed that all descriptions are now aligned with OpenMetrics convention, so it is not blocking HTTP semantic convention stability.

@trask FYI
